src/commands/periscope/helpers/periscopehelper.ts

```diff
@@ -109,51 +115,67 @@ export async function getStorageInfo(
     }
 }
 
+async function extractContainerName(clusterKubeConfig: string): Promise<string | undefined> {
+        const kubectl = await k8s.extension.kubectl.v1;
+
+        if (!kubectl.available) {
+            vscode.window.showWarningMessage(`Kubectl is unavailable.`);
+            return;
```

Comment:
Recommend `return undefined;` to make it clear that the return value is significant.